tests/integration/test_storage_postgresql/test.py

```diff
@@ -836,6 +837,40 @@ def test_fixed_string_type(started_cluster):
     node1.query("DROP TABLE test_fixed_string")
 
 
+def test_parameters_validation_for_postgresql_function(started_cluster):
+    cursor = started_cluster.postgres_conn.cursor()
+
+    def _create_and_fill_table(table):
+        cursor.execute(f"DROP TABLE IF EXISTS {table}")
+        cursor.execute(f"CREATE TABLE {table} (a Integer)")
+        cursor.execute(f"INSERT INTO {table} SELECT 1")
+
+    # Try to do some SQL injection to remove the original table
+    table = f"test_parameters_validation_for_postgresql_function_{uuid.uuid4().hex[:8]}"
+    _create_and_fill_table(table)
+    try:
+        node1.query(
```

Comment:
better to use `assert "expected exception" in node1.query_and_get_error(query)`

Reply:
Done in  [`c31de2d` (#74144)](https://github.com/ClickHouse/ClickHouse/pull/74144/commits/c31de2dd7fe555c6e074e4828ca83d142e802c2b)